tests/prov/test_prov_k8s_cortx_upgrade.py

```diff
@@ -215,6 +185,9 @@ def test_33669(self):
     def test_41953(self):
         """Verify resume of in progress rolling upgrade."""
         process_list = []
+        self.upgrade_obj.retain_solution_file(
```

Comment:
Please rename the function retain_solution_file 
As it modifies the file to newer values and not save the original for further use.

Reply:
Also there are multiple instance using the hardcoded 'solution.yaml'
Please add it as constants and then use it in libs, tests. 
can be found in retain_solution_file